### PR TITLE
[Batch] Management Plane v14.2.0 - 2022-01-01

### PR DESCRIPTION
**Pull request by paterasMSFT**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/batch/Microsoft.Azure.Management.Batch/tests/InMemoryTests/DetectorTests.InMemory.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
...CAgImlkIjogIjMyNjM1MDkxIiwNCiAgICAgICAgInBlc0lkIjogIjE1NjE0IiwNCiAgICAgICAgInR5cGVJZCI6ICJEaWFnbm9zdGljcy5Nb2RlbHNBbmRVdGlscy5BdHRyaWJ1dGVzLlN1cHBvcnRUb3BpYywgRGlhZ25vc3RpY3MuTW9kZWxzQW5kVXRpbHMsIFZlcnNpb249MS4wLjAuMCwgQ3VsdHVyZT1uZXV0cmFsLCBQdWJsaWNLZXlUb2tlbj1udWxsIg0KICAgICAgfSwNCiAgICAgIHsNCiAgICAgICAgImlkIjogIjMyNjM1MDkzIiwNCiAgICAgICAgInBlc0lkIjogIjE1NjE0IiwNCiAgICAgICAgInR5cGVJZCI6ICJEaWFnbm9zdGljcy5Nb2RlbHNBbmRVdGlscy5BdHRyaWJ1dGVzLlN1cHBvcnRUb3BpYywgRGlhZ25vc3RpY3MuTW9kZWxzQW5kVXRpbHMsIFZlcnNpb249MS4wLjAuMCwgQ3VsdHVyZT1uZXV0cmFsLCBQdWJsaWNLZXlUb2tlbj1udWxsIg0KICAgICAgfSwNCiAgICAgIHsNCiAgICAgICAgImlkIjogIjMyNjM1MDk0IiwNCiAgICAgICAgInBlc0lkIjogIjE1NjE0IiwNCiAgICAgICAgInR5cGVJZCI6ICJEaWFnbm9zdGljcy5Nb2RlbHNBbmRVdGlscy5BdHRyaWJ1dGVzLlN1cHBvcnRUb3BpYywgRGlhZ25vc3RpY3MuTW9kZWxzQW5kVXRpbHMsIFZlcnNpb249MS4wLjAuMCwgQ3VsdHVyZT1uZXV0cmFsLCBQdWJsaWNLZXlUb2tlbj1udWxsIg0KICAgICAgfQ0KICAgIF0sDQogICAgImFuYWx5c2lzVHlwZXMiOiBudWxsLA0KICAgICJ0eXBlIjogIkFuYWx5c2lzIiwNCiAgICAic2NvcmUiOiAwLjANCiAgfSwNCiAgImRhdGFzZXQiOiBbXSwNCiAgInN0YXR1cyI6IHsNCiAgICAibWVzc2FnZSI6IG51bGwsDQogICAgInN0YXR1c0lkIjogNA0KICB9LA0KICAiZGF0YVByb3ZpZGVyc01ldGFkYXRhIjogbnVsbCwNCiAgInN1Z2dlc3RlZFV0dGVyYW5jZXMiOiBudWxsDQp9'
```
Comment (wiboris) [inline sdk/batch/Microsoft.Azure.Management.Batch/tests/InMemoryTests/DetectorTests.InMemory.cs:28]: is it ok to check in this key

**Review by wiboris — APPROVED** [review]
The only thing I'm wondering is if we should scrub any keys

File: sdk/batch/Microsoft.Azure.Management.Batch/tests/InMemoryTests/DetectorTests.InMemory.cs (hunk 1 above)
Comment (paterasMSFT) [reply sdk/batch/Microsoft.Azure.Management.Batch/tests/InMemoryTests/DetectorTests.InMemory.cs:28]: Yeah. I pulled it from [the example](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/batch/resource-manager/Microsoft.Batch/stable/2022-01-01/examples/DetectorGet.json) in the public swagger repo.

**Review by dpwatrous — APPROVED** [review]

**Comment by paterasMSFT** [comment]
@fengzhou-msft, would you mind reviewing when you get a moment? We're anxious to merge. Thanks!

**Review by fengzhou-msft — CHANGES_REQUESTED** [review]
Please update version in Microsoft.Azure.Management.Batch.csproj and AssemblyInfo.cs

**Comment by paterasMSFT** [comment]
> Please update version in Microsoft.Azure.Management.Batch.csproj and AssemblyInfo.cs

Done

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**